core-bundle/src/Controller/CaptchaController.php

```diff
@@ -37,6 +37,10 @@ public function __construct(ContaoFramework $framework)
      */
     public function __invoke(Request $request): Response
     {
+        if (!$request->isXmlHttpRequest()) {
+            return new Response($this->getScriptSource(), 200, ['Content-Type' => 'text/javascript', 'Cache-Control' => 'private, max-age=604800']);
```

Comment:
```suggestion
            return new Response($this->getScriptSource(), 200, ['Content-Type' => 'text/javascript', 'Cache-Control' => 'private, max-age=0']);
```

Now that we don‘t check for `no-cache` anymore, the Ajax request would be loaded from the cache, wouldn‘t it?

Reply:
No, the fetch request does not use the cache in our case (that is standardized and documented), it only does not send that information to the server on mobile chrome (I don’t know if it should do that acording to a standard). So caching the javascript is still correct, and also a good thing because it should be fast at setting the captcha to `display: none`.